setup.py

```diff
@@ -4,6 +4,7 @@
 setuptools.setup(name='zwad',
                  packages=['zwad'],
                  install_requires=[
+                     'coniferest @ git+https://github.com/snad-space/coniferest.git@de3112ab0f2b5bea5d7d33887b2e119b1476840e#egg=coniferest-0.0.2',
```

Comment:
Can we use a pypi version?

Reply:
These are the changes: https://github.com/snad-space/coniferest/compare/84bbbdb133025c11f77d0b974b4867969a2ae078...955567356f0c88b11fcbdbdacfcd68addb6f9d6a